<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1428</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="tInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>621</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>671</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_Setup">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP Recv IP Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP Send IP Port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP Recv IP Port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>147</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP Send IP Address:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tSendServer">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="tSendPort">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="tRecvServer">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="tRecvPort">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="bConnect">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_IMU">
     <attribute name="title">
      <string>IMU</string>
     </attribute>
     <widget class="QLabel" name="lPanServo">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pan Servo: 0</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tPanServoMin">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="sPanServo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="tPanServoMax">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="tPanServoTime">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbParallelOperation">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update In Parallel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bPreview">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min %</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max %</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bManualControl">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>510</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min %</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tRollServoTime">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="tRollServoMax">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="sRollServo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
     <widget class="QLabel" name="lRollServo">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Roll Servo: 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>160</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max %</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tRollServoMin">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lTiltServo">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tilt Servo: 0</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tTiltServoTime">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
     <widget class="QSlider" name="sTiltServo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>290</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max %</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tTiltServoMin">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="tTiltServoMax">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>310</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min %</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bAutoControl">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>510</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bPanServoReset">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bRollServoReset">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bTiltServoReset">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>290</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="tMessageInfo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1428</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
